shared/js/background/devbuild.js

```diff
@@ -17,26 +17,31 @@ const { TabState } = require('./classes/tab-state')
 const Wrapper = require('./wrapper.js')
 const { setListContents, getListContents } = require('./message-handlers')
 
-// @ts-ignore - dbg is not a standard property of self.
-self.dbg = {
-    settings,
-    startup,
-    tabManager,
-    Tab,
-    TabState,
-    Wrapper,
-    atb,
-    https,
-    tds,
-    browserWrapper,
-    utils,
-    setListContents,
-    getListContents,
-    companies: Companies,
-    ntts: createNewtabTrackerStatsDebugApi()
-}
+// @ts-ignore
+// RELOADER && require('./devbuild-reloader')
+
+export default function setUpDevbuild () {
```

Comment:
How does this get called?

Reply:
In background.js - I've updated the naming to match the usage there, so it's clearer.